### ISSUE TITLE
[MoM] Spacial Vortex barely does anything (and possible directed_push is bugged?)

### ISSUE THREAD

**Issue by Terrorforge** [CleverRaven/Cataclysm-DDA#69929]
### Describe the bug

The Teleporter power "Spacial Vortex" only pulls enemies 1 tiles, which is borderline useless and clearly not intended. It does this because it uses the area_pull effect which, per MAGIC.md, is restricted to only moving targets 1 tile. For greater effects, you're supposed to use directed_push with a negative value.

But I tried that, and it doesn't work. Changing the effect to directed_push does allow the spell to correctly move enemies more like 3-5 tiles as intended, but even though the values are negative, it always pushes. Making them positive doesn't change this. So either I'm just dumb or the directed_push effect itself isn't working as intended, but either way I don't know how to fix it.

### Attach save file

trivially reproducible

### Steps to reproduce

Cast Spacial Vortex on an enemy, ideally with some kind of marker so you can see how far it moves

### Expected behavior

Spacial Vortex has a meaningful effect

directed_push works the way the documentation says it does

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3693 (22H2)
- Game Version: dac4ed2 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth],
    Mind Over Matter [mindovermatter],
    Bionic Slots [cbm_slots],
    Stats Through Kills [stats_through_kills],
    Massachusetts [MA]
]

**Comment by Standing-Storm** [comment]
It's the latter, unfortunately.  A persistent problem with getting Telekinesis working. 

You're right that Spacial Vortex doesn't work the way I want it to, but none of my attempts using directed_push were better. Sometimes it would move monsters, but in a seemly random direction (and occasionally _towards_ the player!).  Sometimes it would move the monsters but create immobile, invincible "copies" of the monsters who remained in their original locations. Sometimes it did nothing at all. After multiple attempts where nothing worked quite right, I switched it over to area_pull since that at least moved all monsters toward the center of the effect, even if it was only a single tile.

(Wave of Force had the same problems until I reworked it so behind the scenes it just channels Force Shove 12 times on random targets within 1 square of the player.  Unfortunately, that won't work for Spacial Vortex)
